iceoryx_hoofs/source/posix_wrapper/shared_memory_object.cpp

```diff
@@ -79,15 +79,15 @@ SharedMemoryObject::SharedMemoryObject(const SharedMemory::Name_t& name,
         std::cerr << "Unable to create a shared memory object with the following properties [ name = " << name
                   << ", sizeInBytes = " << memorySizeInBytes
                   << ", access mode = " << ACCESS_MODE_STRING[static_cast<uint64_t>(accessMode)]
-                  << ", ownership = " << OWNERSHIP_STRING[static_cast<uint64_t>(ownerShip)]
+                  << ", ownership = " << POLICY_STRING[static_cast<uint64_t>(policy)]
```

Comment:
Should `ownership` be replaced by `policy`? It doesn't seem to be the most suitable word for the possible values of `policy`...

Reply:
Same for line 105